playbooks/katello_stable.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: all
+  roles:
+    - customize_home
```

Comment:
The name of this playbook seems at odds with what it's actually doing. I would assume this would install a stable Katello.

Reply:
This is for the roles that are run after a katello stable VM is provisioned, any suggestions on other names?

Reply:
Off the top of my head `setup_user_development_environment`